api/v1alpha1/olsconfig_types.go

```diff
@@ -211,6 +211,10 @@ type ModelSpec struct {
 	// +kubebuilder:validation:Pattern=`^https?://.*$`
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="URL"
 	URL string `json:"url,omitempty"`
+	// Defines the model's context window size. Default is specific to provider/model.
+	// +kubebuilder:validation:Minimum=0
```

Comment:
@syedriko @asamal4 this minimum cannot be 0 . Do we have a recommendation for this ?

Reply:
I assumed 0 means default and use of https://github.com/openshift/lightspeed-service/blob/1e30de05abf46964441065e6a1f24a63b9ad8e30/ols/constants.py#L76

Reply:
I meant 0 being the default for an int for the `omitempty` purposes.